src/api.rs

```diff
@@ -119,7 +119,8 @@ impl ApiDirectoryMeta {
 pub struct ApiAccount {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub status: Option<String>,
-    pub contact: Vec<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub contact: Option<Vec<String>>,
```

Comment:
An empty vector is perhaps OK? In Rust, an empty vector isn't allocating anything on the heap.

Reply:
Oh. Sorry. That would be weird against the API. I need to think before I comment.

Reply:
Let's merge! Thanks!